sdk/identity/azure-identity/migration_guide.md

```diff
@@ -4,11 +4,25 @@
 
 To encourage best security practices, `azure-identity` does not support JSON- and file-based authentication in the same
 way as `azure-common`. `azure-common` provided factory methods like [`get_client_from_json_dict`][client_from_json] and
-[`get_client_from_auth_file`][client_from_auth_file] to provide credentials as plaintext JSON. Azure credentials, like
-passwords, should not be stored in plaintext whenever possible. You can find information about Azure Active Directory
-(Azure AD) and best practices for authentication in [Azure AD's documentation][azure_ad].
+[`get_client_from_auth_file`][client_from_auth_file] that are no longer available in `azure-identity`.
 
-If it's too difficult to migrate away from JSON- or file-based authentication despite the security risks, you can still
+In `azure-common` you could provide credentials in a JSON dictionary, or from a JSON file:
+```python
+from azure.common.client_factory import get_client_from_json_dict, get_client_from_auth_file
+from azure.keyvault import KeyVaultClient
```

Comment:
KeyVaultClient drags in unrelated migration issues (so does CertificateClient). How about KeyVaultManagementClient?